src/components/AddItemForm.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { format } from 'prettier';
```

Comment:
I think this `prettier` import might be what's causing the checks to fail. I'm not sure. I did run `npm install` and I still get these messages in the terminal. Commenting out this import fixed it.
```  Line 2:10:  'format' is defined but never used  no-unused-vars

./node_modules/prettier/parser-typescript.js
Module not found: Can't resolve '@microsoft/typescript-etw' in '/Users/ForeignFood/Development/collab_lab/tcl-6-smart-shopping-list/node_modules/prettier'```